sktime/forecasting/exp_smoothing.py

```diff
@@ -19,28 +19,39 @@ class ExponentialSmoothing(_StatsModelsAdapter):
 
     Parameters
     ----------
-    trend : str{"add", "mul", "additive", "multiplicative", None}, optional
-    (default=None)
-        Type of trend component.
-    damped_trend : bool, optional (default=None)
+    trend : str or None, default=None
+        Type of trend component.Takes one of 
+        {"add", "mul", "additive", "multiplicative", None}     
+    damped_trend : bool, default=False
```

Comment:
@RHKuhns suggested having options listed directly in the type, i.e.
```suggestion
    trend : {"add", "mul", "additive", "multiplicative", None}, default=None
        Type of trend component.
    damped_trend : bool, default=False
```